server/src/tenant/schema-builder/graphql-types/input/filter-is-enum-filter.type.ts

```diff
@@ -0,0 +1,9 @@
+import { GraphQLEnumType } from 'graphql';
+
+export const FilterIsEnumType = new GraphQLEnumType({
+  name: 'FilterIs',
+  values: {
+    PENDING: { value: 'PENDING' },
```

Comment:
Are those test values? Should we include this filter in this PR?

Reply:
@Weiko Don't really know the goal of that for now, but it's in what `pg_graphql` is generating:
https://supabase.github.io/pg_graphql/example_schema/